### ISSUE TITLE
QuillImageDropAndPaste.ImageData is not a constructor

### ISSUE THREAD

**Issue by nmitra6** [chenjuneking/quill-image-drop-and-paste#27]
Also, one more issue to instantiate an object out of ImageData class. I am using Nuxt (Vue) framework and when importing this class inside plugin, error is thrown as QuillImageDropAndPaste.ImageData is not a constructor.

const ImageData = QuillImageDropAndPaste.ImageData

 But if you declare the class as an  expression, this error goes away. So if you declare as 

ImageData = class {

	constructor(dataUrl, type) {
		this.dataUrl = dataUrl
		this.type = type
	}
 // rest of the codes
}

then that error is not thrown. I would request you to incorporate this modification in the next minor version.
Also if you change the name 'ImageData' it would be better as browsers have builtin ImageData class/method.

**Comment by chenjuneking** [comment]
fixed on v1.2.5.

try this:
```
import { ImageData as QuillImageData } from 'quill-image-drop-and-paste'
```

In order to avoid the changes affecting the previous version, I decided not to change the name of `ImageData`, you could rename the `ImageData` as above.

If you have other questions, please feel free to let me know, thanks!

**Comment by nmitra6** [comment]
Thank you Mr Chen for the upgrade. I will try the new version. With a little change in the coding, it is possible to extend this good module to work universally with any html element, let alone Quill. Hope you will consider that in future version

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows
